backend/src/main/java/com/wootech/dropthecode/domain/TeacherProfile.java

```diff
@@ -29,9 +29,9 @@
     @OneToMany(mappedBy = "teacherProfile", fetch = FetchType.LAZY, cascade = CascadeType.REMOVE)
     private final Set<TeacherSkill> skills = new HashSet<>();
 
-    private final Integer sumReviewCount = 0;
+    private Integer sumReviewCount = 0;
 
-    private final Double averageReviewTime = (double) 0;
+    private Double averageReviewTime = (double) 0;
```

Comment:
0으로 초기화 하는 경우 그냥 primitive type을 사용하는 건 어떤가요?
```suggestion
    private double averageReviewTime;
```

Reply:
위와 똑같이 적용했습니다!

Reply:
엇 오히려 averageReviewTime은 일단은 0으로 초기화하긴 했지만, null일 수도 있지 않을까요? 리뷰를 한 번도 안했으면?